sample-app/src/main/java/com/splunk/android/sample/FirstFragment.java

```diff
@@ -120,9 +121,11 @@ public void onFailure(@NonNull Call call, @NonNull IOException e) {
 
                 @Override
                 public void onResponse(@NonNull Call call, @NonNull Response response) {
-                    int responseCode = response.code();
-                    httpResponse.postValue("" + responseCode);
-                    workflow.end();
+                    try (ResponseBody body = response.body()) {
+                        int responseCode = response.code();
+                        httpResponse.postValue("" + responseCode);
+                        workflow.end();
+                    }
```

Comment:
I also snuck this change in since okhttp was yelling at me in the logs while I was testing things.